helix-core/src/main/java/org/apache/helix/controller/rebalancer/topology/Topology.java

```diff
@@ -43,11 +43,11 @@
  */
 public class Topology {
   private static Logger logger = LoggerFactory.getLogger(Topology.class);
+
   public enum Types {
-    ROOT,
-    ZONE,
-    INSTANCE
+    ROOT, ZONE, INSTANCE
```

Comment:
Is this our new style to format a enum types?

Reply:
I think auto formatter changes this..